src/sql/workbench/browser/designer/designer.ts

```diff
@@ -91,19 +95,22 @@ export class Designer extends Disposable implements IThemable {
 	private _tableCellEditorFactory: TableCellEditorFactory;
 	private _propertiesPane: DesignerPropertiesPane;
 	private _buttons: Button[] = [];
```

Comment:
since you have removed the buttons and replaced them with actionbar, do we still need the _buttons variable?